SamplePlugin/signing_key.xml

```diff
@@ -0,0 +1,16 @@
+<RSAKeyValue>
```

Comment:
having this file be public defeats the purpose :)

Reply:
Yeah, it's just there for testing 👀